### DISCUSSION TITLE
Cannot receive private channel messages through Laravel-Octane.

### DISCUSSION THREAD

**Discussion by nftaverse-admin** [qruto/laravel-wave#46] [Q&A]
"qruto/laravel-wave": "^0.7.1",
"laravel/framework": "10.31",
"laravel/octane": "2.1.1"

In Laravel Octane, if using a public channel, it works fine. However, if a private channel is used, the front-end does not receive the message successfully. I switched back to Nginx with PHP-FPM and it worked fine.

Is Laravel Octane still not supported? If yes, is it being adapted?

**Comment by slavarazum** [comment]
Laravel Octane supported since [0.8.0](https://github.com/qruto/laravel-wave/releases/tag/0.8.0) version.
